pkg/lvm/lvm_util.go

```diff
@@ -119,3 +120,32 @@ func GetVolumeDevPath(vol *apis.LVMVolume) (string, error) {
 
 	return dev, nil
 }
+
+// builldVolumeResizeArgs returns resize command for the lvm volume
+func buildVolumeResizeArgs(vol *apis.LVMVolume) []string {
+	var LVMVolArg []string
+
+	dev := DevPath + vol.Spec.VolGroup + "/" + vol.Name
+	size := vol.Spec.Capacity + "b"
+
+	LVMVolArg = append(LVMVolArg, dev, "-L", size, "-r")
+
+	return LVMVolArg
+}
+
+// ResizeLVMVolume resizes the volume
+func ResizeLVMVolume(vol *apis.LVMVolume, mountpath string) error {
+	volume := vol.Spec.VolGroup + "/" + vol.Name
+
+	args := buildVolumeResizeArgs(vol)
+	cmd := exec.Command(LVExtend, args...)
+	out, err := cmd.CombinedOutput()
+
+	if err != nil {
+		klog.Errorf(
+			"lvm: could not resize the volume %v cmd %v error: %s", volume, args, string(out),
```

Comment:
```suggestion
			"lvm: could not resize the volume %v cmd %v error: %s", volume, cmd, string(out),
```
`cmd` should be used here instead of `args`

Reply:
args are just fine as the command will be `lvcreate args`, we know the lvcreate command as the msg says that.